bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -69,9 +72,7 @@ $(SIM_DIR)/cce_ucode.bin: $(SIM_DIR)/cce_ucode.mem
 	cd $(@D); python3 $(UCODE2BOOT) --ucode=$< --path=$(@D) | $(XXD) -r -p > $@
 
 NBF_INPUTS ?= --ncpus=$(NCPUS) --ucode=cce_ucode.mem
-ifeq ($(PRELOAD_MEM_P), 0)
```

Comment:
We need preloading to get reasonable simulation time for parallel cosim of Linux. Why are we removing it totally?

Reply:
mistake, reverted